### PR TITLE
Adding new Resources SDK -  StandbyPool

### PR DESCRIPTION
**Pull request by fangru2023**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/standbypool/Azure.ResourceManager.StandbyPool/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure StandbyPool management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/standbypool/Azure.ResourceManager.StandbyPool/README.md:3]: Add description of your service.

File: sdk/standbypool/Azure.ResourceManager.StandbyPool/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/standbypool/Azure.ResourceManager.StandbyPool/CHANGELOG.md:3]: When do you want to release the SDK? Please specify the date by replacing the `Unreleased` with an expected data, the format is `yyyy-mm-dd`.

Comment (fangru2023) [reply sdk/standbypool/Azure.ResourceManager.StandbyPool/CHANGELOG.md]: Thanks for reminding, updated.

File: sdk/standbypool/Azure.ResourceManager.StandbyPool/README.md (hunk 1 above)
Comment (fangru2023) [reply sdk/standbypool/Azure.ResourceManager.StandbyPool/README.md]: added.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**